site.yml

```diff
@@ -21,7 +21,6 @@
       - job-dsl
       - github-api
       - github-oauth
-      - parameterized-trigger
```

Comment:
This broke https://github.com/alphagov/multicloud-deploy/pull/26 can we add back in?